target/linux/ramips/dts/mt7621_raisecom_msg1500-x-00.dts

```diff
@@ -150,13 +176,3 @@
 		function = "gpio";
 	};
 };
-
-&factory {
-	compatible = "nvmem-cells";
```

Comment:
I think this node can be removed because the mt76 driver always automatically get the WLAN MAC from `factory@0x04`.

Reply:
Without this, mt76 will read factory@0xa as radio1's mac address. And https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac#L85 only fixes the mac address of radio0.
So this `nvmem-cells = <&macaddr_factory_4>;` can't be removed.